en/ui/widget-align.md

```diff
@@ -73,7 +73,16 @@ When inputting the margin value when opening alignment direction, Widget can use
 
 Making use of the percentage alignment distance, we can create UI elements that can zoom in/out infinitely according to the size of the screen. Exerting your imagination, fitting a thousand types of Android phones with one set of resources will be a piece of cake!
 
+## Update alignment and optimization strategies for every frame at runtime
 
----
+Widget component is generally used to locate the position of each element when the scene is initialized on the target device, but once the scene is initialized, we often do not need to use the Widget component for alignment. The `alignOnce` property is used to ensure that the Widget component only performs alignment and positioning at initialization, and no longer consumes time for alignment at runtime.
```

Comment:
use the Widget component for alignment -> use the Widget to update alignment persistently

Reply:
spends time for alignment at runtime.